libcxx/include/__config

```diff
@@ -167,6 +167,11 @@
 // The implementation moved to the header, but we still export the symbols from
 // the dylib for backwards compatibility.
 #    define _LIBCPP_ABI_DO_NOT_EXPORT_TO_CHARS_BASE_10
+// Same memory by providing the allocator more freedom to allocate the most
```

Comment:
```suggestion
// Save memory by providing the allocator more freedom to allocate the most
```

Reply:
lol sorry this snuck back in after I broke broke the last PR.